### PR TITLE
python310Packages.cryptography: don't depend on pytest-benchmark

### PR DESCRIPTION
**Pull request by dotlambda**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/cryptography/default.nix [hunk 1]
```diff
@@ -79,7 +82,9 @@ buildPythonPackage rec {
     "--disable-pytest-warnings"
   ];
 
-  disabledTestPaths = lib.optionals (stdenv.isDarwin && stdenv.isAarch64) [
+  disabledTestPaths = [
+    "tests/bench"
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/cryptography/default.nix:86]:
```suggestion
    # save compute by not running benchmarks
    "tests/bench"
```

File: pkgs/development/python-modules/pytest-benchmark/default.nix [hunk 2]
```diff
@@ -20,28 +29,52 @@ buildPythonPackage rec {
...
+    "test_rendering"
+  ];
+
   meta = with lib; {
+    changelog = "https://github.com/ionelmc/pytest-benchmark/blob/${src.rev}/CHANGELOG.rst";
     description = "Pytest fixture for benchmarking code";
     homepage = "https://github.com/ionelmc/pytest-benchmark";
     license = licenses.bsd2;
-    maintainers = with maintainers; [ costrouc ];
+    maintainers = with maintainers; [ costrouc dotlambda ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pytest-benchmark/default.nix:78]:
```suggestion
    maintainers = with maintainers; [ dotlambda ];
```
costrouc is MIA most of the time anyway

**Review by SuperSandro2000 — APPROVED** [review]

**Comment by dotlambda** [comment]
FYI Cryptography's benchmarks also weren't run before this change.

**Comment by SuperSandro2000** [comment]
I am going to create a PR which disables those tests https://github.com/ionelmc/pytest-benchmark/issues/231

**Comment by SuperSandro2000** [comment]
#230170

**Pull request merged**